SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -625,10 +625,17 @@
 run2_HCAL_2017.toModify( g4SimHits, HCalSD = dict( TestNumberingScheme = True ) )
 
 ##
-## Disable Castor from Run 3
+## Disable Castor from Run 3, enable PPS
 ##
 from Configuration.Eras.Modifier_run3_common_cff import run3_common
 run3_common.toModify( g4SimHits, CastorSD = dict( useShowerLibrary = False ) ) 
+run3_common.toModify( g4SimHits, LHCTransport = cms.bool(True) )
+
+##
+## Disable PPS from Run 3 PbPb runs
+##
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+pp_on_PbPb_run3.toModify( g4SimHits, LHCTransport = false )
```

Comment:
My bad, sorry Vladimir:  "False" should have been written with capital "F"
```suggestion
pp_on_PbPb_run3.toModify( g4SimHits, LHCTransport = False )
```